### ISSUE TITLE
UHAF - Utah Homeowner's Assistance Fund App 

### ISSUE THREAD

**Issue by DenisePeterson** [agrc/planning-queue#280]
### Benefit

Create an interactive Dashboard for the [Utah Homeowners Assistance Fund](https://homeownersassistance.utah.gov/) to act as a resource for statewide information regarding assistance approved and paid, households assisted, average assistance amounts, etc…

### Issue Reference

refs #

**Comment by DenisePeterson** [comment]
Support and maintenance for UHAF app

**Comment by DenisePeterson** [comment]
2024-02-13:  Incorporated new data into dashboard.

**Comment by nathankota** [comment]
@DenisePeterson it looks like the app was successfully created and you are in data update/app maintenance mode? Do you think it is appropriate to close this issue/task and create a new one for Q4 for updates and maintenance, if that type of work is expected to happen?

**Comment by DenisePeterson** [comment]
@nathankota Yes, I plan to close this and create a new one for Q4. I'll be updating it monthly as I receive updated reports from UHAF.

**Comment by DenisePeterson** [comment]
Closed issue and opened new for FY24 - Q4.
